### PR TITLE
Initial add of SSTU SuperDraco and the new XLR81

### PR DESCRIPTION
**Pull request by SirKeplan**
~~Don't merge yet, more to come.~~

### CODE REVIEW

**Comment by SirKeplan** [comment]
Should be good now, though i'd like to add the Vacuum SuperDraco too, anyone have a good estimate of what performance it would/should have?

**Comment by stratochief66** [comment]
I would suggest leaving the vacuum version for a later PR. What situation would the vacuum version be used in? Mars retro-propulsion?

**Comment by SirKeplan** [comment]
> What situation would the vacuum version be used in? Mars retro-propulsion? 
don't know about Mars retro propulsion, would the bell be strong enough in real life?. Anyway the vacuum version here is just a hypothetical vacuum version of a SuperDraco, not an engine that's in actual development, though it could be good item to increase the low number of throttleable lander engines we have in RO.

File: GameData/RealismOverhaul/RealPlume_Configs/SSTU/SSTU-SC-ENG-LR81-8048.cfg [hunk 1]
```diff
@@ -0,0 +1,43 @@
+//	==================================================
+//	SuperDraco plume configuration.
```
Comment (ahmedcharles) [inline GameData/RealismOverhaul/RealPlume_Configs/SSTU/SSTU-SC-ENG-LR81-8048.cfg]: This isn't the superdraco?

File: GameData/RealismOverhaul/RealPlume_Configs/SSTU/SSTU-SC-ENG-LR81-8096.cfg [hunk 2]
```diff
@@ -0,0 +1,43 @@
+//	==================================================
+//	SuperDraco plume configuration.
```
Comment (ahmedcharles) [inline GameData/RealismOverhaul/RealPlume_Configs/SSTU/SSTU-SC-ENG-LR81-8096.cfg]: This isn't the superdraco?

File: GameData/RealismOverhaul/RealPlume_Configs/SSTU/SSTU-SC-ENG-LR81-8048.cfg (hunk 1 above)
Comment (SirKeplan) [inline GameData/RealismOverhaul/RealPlume_Configs/SSTU/SSTU-SC-ENG-LR81-8048.cfg]: @ahmedcharles ahah, i left the wrong comment in the files, thanks for noticing

**Pull request merged**